Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -500,11 +463,11 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
     open func showTypingIndicator(typingMembers: [_ChatChannelMember<ExtraData.User>]) {
         if typingIndicatorView.isHidden {
             Animate {
-                self.collectionView.contentInset.bottom += self.typingIndicatorViewHeight
-                self.collectionView.scrollIndicatorInsets.bottom += self.typingIndicatorViewHeight
+                self.tableView.contentInset.top += self.typingIndicatorViewHeight
+                self.tableView.scrollIndicatorInsets.top += self.typingIndicatorViewHeight
             }
 
-            if collectionView.isLastCellVisible {
+            if tableView.isLastCellVisible {
                 scrollToMostRecentMessage()
```

Comment:
Can we double check we didn't lose the fix from https://github.com/GetStream/stream-chat-swift/pull/1210 ?

Reply:
we didn't 👍